internal/config/config.go

```diff
@@ -34,22 +35,37 @@ const (
 	ClientTokenEnvVar    = "EQUINIX_API_TOKEN"
 	ClientTimeoutEnvVar  = "EQUINIX_API_TIMEOUT"
 	MetalAuthTokenEnvVar = "METAL_AUTH_TOKEN"
+
+	// This is the initial env var for HTTP logging for backwards
+	// compatibility.  We could now introduce service-specific
+	// variables, if desired, for more granular control of logging
+	HTTPLoggingEnvVar = "TF_LOG"
```

Comment:
Using `TF_LOG` to control the log level for all of the new subsystem loggers replicates prior behavior but may not be desirable.  Now that we have these loggers in place, we could add support for one or more of `TF_LOG_PROVIDER_EQUINIX` (to specify log level for the whole provider) or `TF_LOG_EQUINIX_<service>` (to allow more granular configuration of provider logging).